Modules/_testcapi/getargs.c

```diff
@@ -447,6 +448,16 @@ getargs_p(PyObject *self, PyObject *args)
     return PyLong_FromLong(value);
 }
 
+static PyObject *
+getargs_p_star(PyObject *self, PyObject *args)
+{
+    bool value;
+    if (!PyArg_ParseTuple(args, "p*", &value)) {
+        return NULL;
+    }
+    return PyLong_FromLong((int) value);
```

Comment:
```suggestion
    return PyLong_FromLong((int)value);
```

Casts don't have a space the parenthesis (at least not in this file). In the `getargs.c` file it seems there are spaces. So let's be consistent.